### PR TITLE
Updates for 0.25 RYF config

### PR DESCRIPTION
**Pull request by anton-seaice**
Contributes to https://github.com/ACCESS-NRI/access-om2-configs/issues/17

This adds 0.25 degree RYF config

### CODE REVIEW

**Comment by anton-seaice** [comment]
I don't understand this failure:

`
AssertionError: Expect collate executable set to mppnccombine-fast
`

It's set to 
`  exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/mom5-git.2023.11.09=2023.11.09-ewcdbrfukblyjxpkhd3mfkj4yxfolal4/bin/mppnccombine.spack
`

Which I believe is correct?

**Comment by aidanheerdegen** [comment]
> Which I believe is correct?

https://github.com/ACCESS-NRI/access-om2-configs/pull/68#issuecomment-2044055563

**Review by aidanheerdegen — APPROVED** [review]
LGTM

File: config.yaml [hunk 1]
```diff
@@ -51,8 +56,8 @@ submodels:
       model: cice5
       exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/cice5-git.2023.10.19=2023.10.19-rh3xfkrgajya3ghtliacuhlx3pgvrzqs/bin/cice_auscom_1440x1080_48x40_480p.exe
```
Comment (aidanheerdegen) [inline config.yaml:57]:
```suggestion
      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/cice5-git.2023.10.19=2023.10.19-v3zncpqjj2gyseudbwiudolcjq3k3leo/bin/cice_auscom_1440x1080_48x40_480p.exe
```

Update to the latest ACCESS-OM2 release.

File: config.yaml [hunk 2]
```diff
@@ -22,7 +27,7 @@ input:
 submodels:
     - name: atmosphere
       model: yatm
-      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/libaccessom2-git.2023.10.26=2023.10.26-ieiy3e7hidn4dzaqly3ly2yu45mecgq4/bin/yatm.exe
+      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/libaccessom2-git.2023.10.26=2023.10.26-ltfg7jcn6t4cefotvj3kjnyu5nru26xo/bin/yatm.exe
```
Comment (aidanheerdegen) [inline config.yaml:30]: This is correct but the mom exe needs to be updated too ([can't comment on unchanged code block](https://github.com/github/roadmap/issues/347), a real missing feature there GitHub)

```yaml
      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/mom5-git.2023.11.09=2023.11.09-qji4nlmr6utrribaiyhewe4je6mifguz/bin/fms_ACCESS-OM.x
```

File: metadata.yaml [hunk 3]
```diff
@@ -1,32 +1,34 @@
...
-  Run with JRA55-do v1.4.0 RYF9091 (1 May 1990 - 30 April 1991) repeated forcing
-  with all solid runoff converted to liquid runoff with no heat transfer.
-  Spin up starts from a nominal year of 1 Jan 1900.
+description: "1 degree ACCESS-OM2 global model configuration under the RYF9091 Repeat\n
+  Year Forcing strategy outlined by Stewart et al. (2020), \nhttps://doi.org/10.1016/j.ocemod.2019.101557.\n
+  The configuration is based on that described in Kiss et al. (2020),\nhttps://doi.org/10.5194/gmd-13-401-2020,
+  but with many improvements.\nInitial conditions are WOA13v2 potential temperature
+  and practical salinity.\nRun with JRA55-do v1.4.0 RYF9091 (1 May 1990 - 30 April
+  1991) repeated forcing\nwith all solid runoff converted to liquid runoff with no
+  heat transfer.\nSpin up starts from a nominal year of 1 Jan 1900."
```
Comment (aidanheerdegen) [inline metadata.yaml:10]: This is the 1 degree description.

**Review by aidanheerdegen — CHANGES_REQUESTED** [review]
Thanks. A few things to fix.

**Review by aidanheerdegen — DISMISSED** [review]
LGTM

**Comment by anton-seaice** [comment]
> LGTM

Wrong answer ! :)

**Review by aidanheerdegen — APPROVED** [review]
Great, thanks for catching the catch I didn't.

**Pull request merged**